Cargo.toml

```
@@ -13,3 +13,7 @@ ariel-os-boards = { path = "build/imports/ariel-os/src/ariel-os-boards" }
 [dev-dependencies]
 embedded-test = { version = "0.6.1", features = ["ariel-os"] }
 {% endif -%}
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = [
+  'cfg(context, values(any()))',
+] }
```

Can you make sure we have a blank line between sections, and a single new line character at EOF, GitHub shows there is none?

```suggestion

[lints.rust]
unexpected_cfgs = { level = "warn", check-cfg = [
  'cfg(context, values(any()))',
] }
```

---

sure, should be fixed now